### PR TITLE
API Configuration-management

### PR DESCRIPTION
**Pull request by mikimartin88**
Good luck ;)

### CODE REVIEW

File: hotelx/cfg/object_UpdateStatus.graphql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+type UpdateStatus{
```
Comment (arquio) [inline hotelx/cfg/object_UpdateStatus.graphql]: implements Node

File: hotelx/cfg/object_UpdateStatus.graphql [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  # New value set
+  data: UpdateStatusData
+
+  # Errors that abort services
+  errors: [Error!]
```
Comment (arquio) [inline hotelx/cfg/object_UpdateStatus.graphql]: deprecated!

File: hotelx/cfg/object_UpdateStatus.graphql [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+  # Potentially harmful situations or errors, but ones that will not forcibly abort the service
+  warnings: [Warning!]
```
Comment (arquio) [inline hotelx/cfg/object_UpdateStatus.graphql]: deprecated

**Comment by arquio** [comment]
@mikimartin88 did you fix its?  could you push it?

**Comment by mikimartin88** [comment]
Hi @arquio ,

Could you please check it?

**Comment by mikimartin88** [comment]
Hi @arquio ,

do you need more changes?

File: hotelx/cfg/inputs/input_HotelXDefaultSettingsInput.graphql [hunk 4]
```diff
@@ -0,0 +1,8 @@
+input HotelXDefaultSettingsInput {
+  # Indicates the group where you will set the default settings
+  group: String!
+  # Indicates to the client's name where you will set the default settings, it's optional because you can set settings only to the group level.
+  clientName: String
```
Comment (arquio) [inline hotelx/cfg/inputs/input_HotelXDefaultSettingsInput.graphql]: ID mejor,

File: hotelx/cfg/inputs/input_HotelXUpdateStatusInput.graphql [hunk 5]
```diff
@@ -0,0 +1,7 @@
+input HotelXUpdateStatusInput{
+  group: ID!
+  access: ID
+  supplier: ID
+  client: ID
```
Comment (arquio) [inline hotelx/cfg/inputs/input_HotelXUpdateStatusInput.graphql]: client name or client code? are differents...

**Comment by mikimartin88** [comment]
Hi @arquio ,

could you please check it?

**Comment by arquio** [comment]
@mikimartin88 : Type "HotelXBusinessRulesType" not found.

**Pull request closed without merging**